scripts/docker-build

```diff
@@ -70,31 +78,48 @@ function build() {
     set +x
 
     echo "Copying build output from $tmp_outdir to $PWD"
-    cp "$tmp_outdir/internet_identity.wasm" .
+    cp "$tmp_outdir/$canister.wasm" .
 
     echo "Removing $tmp_outdir"
     rm -rf "$tmp_outdir"
 }
 
 # ARGUMENT PARSING
 
+CANISTER=()
 while [[ $# -gt 0 ]]
 do
     case $1 in
-        --help)
+        -h|--help)
             title
             usage
             help
             exit 0
             ;;
+        --internet-identity)
+            CANISTER+=("internet_identity")
+            shift
+            ;;
+        --archive)
+            CANISTER+=("archive")
+            shift
+            ;;
         *)
             echo "ERROR: unknown argument $1"
             usage
             echo
-            echo "Use 'release --help' for more information."
+            echo "Use 'docker-build --help' for more information"
             exit 1
             ;;
     esac
 done
 
-build
+# build everything by default
```

Comment:
this only builds II though, right?

Reply:
Ah, yes. I have also fixed the same error in the `build` script.